pkg/cmd/render/render.go

```diff
@@ -0,0 +1,204 @@
+/*
+Copyright 2019 The OpenShift Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package render
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	corev1 "k8s.io/api/core/v1"
+	yaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	minterv1 "github.com/openshift/cloud-credential-operator/pkg/apis/cloudcredential/v1"
+	assets "github.com/openshift/cloud-credential-operator/pkg/assets/bootstrap"
+	"github.com/openshift/cloud-credential-operator/pkg/operator/utils"
+)
+
+const (
+	podTemplate = `apiVersion: v1
+kind: Pod
+metadata:
+  name: cloud-credential-operator
+  namespace: openshift-cloud-credential-operator
+spec:
+  containers:
+  - command:
+    - /root/manager
```

Comment:
point to new /usr/bin/cloud-cred-operator binary name

Reply:
good one!